composables/usePrefix.ts

```diff
@@ -2,26 +2,16 @@ import { DEFAULT_PREFIX } from '@kodadot1/static'
 import type { Prefix } from '@kodadot1/static'
 import { useWalletStore } from '@/stores/wallet'
 import { useAssetsStore } from '@/stores/assets'
-import { getAvailablePrefix } from '@/utils/chain'
```

Comment:
Could you also remove the function definition from utils ? `getAvailablePrefix`

Reply:
oohh thank you, removed